src/Campaigns/Actions/PreventAddFormWithoutCampaign.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+use Give\Campaigns\Models\Campaign;
+
+/**
+ * @unreleased
+ */
+class PreventAddFormWithoutCampaign
```

Comment:
Can we rename this to reflect what it does (in addition to why)? Maybe something like `RedirectLegacyCreateFormRequest`?

Reply:
Or, where is this needed? Shouldn't the use cases just be removed?

Reply:
Resolved in 20a371b09bc913af8830b274a02713088a3867ca